presto-docs/src/main/sphinx/release/release-0.216.rst

```diff
@@ -0,0 +1,75 @@
+=============
+Release 0.216
+=============
+
+General Changes
+---------------
+
+* Fix correctness issue for operations on timestamps in certain time zones (:issue:`11891`).
+* Fix correctness issue for :func:`array_intersect` and :func:`array_distinct` when input contains
+  both zeros and nulls.
+* Fix ``count(*)`` aggregation on empty relation when ``optimize_mixed_distinct_aggregation`` is enabled.
+* Improve the table scan performance for structured types.
```

Comment:
remove "the" and change to "structured" to "structural"

Is this an engine change that applies to all connectors?

Reply:
> Is this an engine change that applies to all connectors?

I think this could be mine. Originally it was: ```Improve performance of table scan on structured types by removing unnecessary memory accounting.```

Yes. this applies to all connectors